packages/components/CHANGELOG.md

```diff
@@ -16,6 +16,10 @@ The following icons are used in commit messages and this changelog.
 
 ## 🚀 NEXT
 
+### 📚 Documentation
+
+- #17: Adjusted package.json author field
```

Comment:
I'd rather document only things relvant for the user of this library. All other stuff is internal and not of interest. At least thats how we handles the CHANGELOG before.

Reply:
As I see it, a changelog should provide a list of all changes. This also means we should adjust it when there are internal values that have changed or Documentation. Should be discussed.